routes/v1.js

```diff
@@ -456,39 +465,49 @@ api.declare({
     return;
   }
 
+  // Parse timestamps
+  let created = new Date(taskDef.created);
+  let deadline = new Date(taskDef.deadline);
+  let expires = new Date(taskDef.expires);
+
   // Insert entry in deadline queue
-  var deadline = new Date(taskDef.deadline);
   await this.queueService.putDeadlineMessage(taskId, deadline);
 
   // Try to create Task entity
   try {
+    let runs = [];
```

Comment:
A note for myself for later (this may be wrong): It looks like we always create tasks no matter whether or not they're ready to run.  `runs` will just be empty until they're ready to actually run.